lib/exercism_config/setup_dynamodb_client.rb

```diff
@@ -0,0 +1,31 @@
+module ExercismConfig
+  class SetupDynamoDBClient
+    include Mandate
+
+    def call
+      aws_settings = GenerateAwsSettings.().merge(
+        endpoint: config_endpoint,
+
+        # We don't want a profile for this AWS service
+        # as we run it locally. But we do normally, so 
+        # we locally override this here.
```

Comment:
How do you know you're running it locally?

Reply:
The comment was badly written. For local AWS we use profiles. For production, we don't use profiles (because the machines come pre-loaded with the right permissions). But for dynamodb, because we're running the service locally, we don't need a profile as we're not connecting to the real AWS.

I'll update the comments in a later PR.

Reply:
Okay cool.